packages/components/src/checkbox/src/Checkbox.module.css

```diff
@@ -0,0 +1,212 @@
+    --hop-Checkbox-icon-color-disabled: var(--hop-neutral-icon-disabled);
+
+    /* Focus */
+    --hop-Checkbox-box-border-color-focus: var(--hop-neutral-border-hover);
+    --hop-Checkbox-box-background-color-focus: var(--hop-neutral-surface-hover);
+    --hop-Checkbox-box-focus-ring-color: var(--hop-primary-border-focus);
+    --hop-Checkbox-box-border-color-checked-focus: var(--hop-neutral-border-active);
+    --hop-Checkbox-box-background-color-checked-focus: var(--hop-neutral-surface-active);
+    --hop-Checkbox-text-color-focus: var(--hop-neutral-text);
+    --hop-Checkbox-icon-color-focus: var(--hop-neutral-icon);
+
+    /* Invalid  */
+    --hop-Checkbox-box-border-color-invalid: var(--hop-danger-border-strong);
+    --hop-Checkbox-box-background-color-invalid: var(--hop-neutral-surface);
+    --hop-Checkbox-box-border-color-hover-invalid: var(--hop-danger-border-strong);
+    --hop-Checkbox-box-background-color-hover-invalid: var(--hop-danger-surface);
+    --hop-Checkbox-box-border-color-focus-invalid: var(--hop-danger-border-strong);
+    --hop-Checkbox-box-background-color-focus-invalid: var(--hop-danger-surface);
+    --hop-Checkbox-box-border-color-checked-invalid: var(--hop-danger-border-strong);
+    --hop-Checkbox-box-background-color-checked-invalid: var(--hop-danger-surface-strong);
+    --hop-Checkbox-check-color-invalid: var(--hop-danger-icon-strong);
+    --hop-Checkbox-text-color-invalid: var(--hop-danger-text-weak);
+    --hop-Checkbox-icon-color-invalid: var(--hop-danger-icon-weak);
+
+    /* Small */
+    --hop-Checkbox-box-sm-inline-size: 1rem;
+    --hop-Checkbox-text-sm-top-offset: calc((var(--box-inline-size) - (var(--hop-body-sm-line-height) * var(--hop-body-sm-font-size))) / 2); 
+    --hop-Checkbox-text-sm-margin-inline-start: calc(var(--hop-space-inline-sm) - var(--hop-space-inline-xs));
+
+    /* Medium */
+    --hop-Checkbox-box-md-inline-size: 1.5rem;
+    --hop-Checkbox-text-md-top-offset: calc((var(--box-inline-size) - (var(--hop-body-md-line-height) * var(--hop-body-md-font-size))) / 2); 
+    --hop-Checkbox-text-md-margin-inline-start: calc(var(--hop-space-inline-md) - var(--hop-space-inline-xs));
+
+    /* Internal variable */
+    --box-background-color: var(--hop-Checkbox-box-background-color);
+    --box-border-color: var(--hop-Checkbox-box-border-color);
+    --box-border-radius: var(--hop-shape-rounded-sm);
+    --check-color: var(--hop-Checkbox-check-color);
+    --check-display: var(--hop-Checkbox-check-display);
+    --cursor: pointer;
+    --text-color: var(--hop-Checkbox-text-color);
+    --icon-color: var(--hop-Checkbox-icon-color);
+    --description-color: var(--hop-Checkbox-description-color);
+
+    align-items: start;
+    column-gap: var(--hop-space-inline-xs);
+    cursor: var(--cursor);
+    display: flex;
+    inline-size: max-content;
+    justify-content: start; 
+    max-inline-size: 100%;
+}
+
+.hop-Checkbox,
```

Comment:
yes I will add border-box

Reply:
actually I'm confused by this comment, I have them in each of my components, I'm guessing you meant that we should have this in a global css as well?

Reply:
Yup I wonder if we should have it globally. Although setting it only when necessary is another strategy.

Reply:
hmm personally, I think it makes sense to set it per component. I'm not sure how we would set it globally, making sure it applies to all our components when it's in use, but not to the user's css. We'd need like a global component css and a global component class or something. I think it may be more complicated than it's worth, but maybe @alexasselin008 has an idea about this.

Reply:
We never want to apply styling to elements we don't control from inside Orbiter. That's why we don't ship with a CSS reset.

We should add  `box-sizing: border-box;` on all our components. 



ideally, we also avoid the `*` in the selector, and only apply box-sizing to hop-Checkbox. 

so i would delete this selector and move box-sizing on the root selector hop-Checkbox

Reply:
the * selector just says to apply it on each element inside the checkbox. without this, we may still have some weird side effects if the application using this component has some other styles using content-box.